data/gpx_test_data/osmand2.gpx

```diff
@@ -0,0 +1,45 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes' ?>
+<gpx version="1.1" creator="OsmAnd~ 4.4.7" xmlns="http://www.topografix.com/GPX/1/1" xmlns:osmand="https://osmand.net" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd">
+  <metadata>
+    <name>favorites-Testowa</name>
+    <time>2023-06-26T12:47:07Z</time>
+  </metadata>
+  <wpt lat="52.0027434" lon="19.1427702">
+    <ele>114</ele>
+    <time>2023-06-26T12:46:49Z</time>
+    <name>Jednostka Wojskowa Nr 4393 i 4395</name>
+    <type>Testowa</type>
+    <extensions>
+      <osmand:amenity_subtype>military_landuse</osmand:amenity_subtype>
+      <osmand:osm_tag_icao>EPLY</osmand:osm_tag_icao>
+      <osmand:amenity_name>Jednostka Wojskowa Nr 4393 i 4395</osmand:amenity_name>
+      <osmand:amenity_type>military</osmand:amenity_type>
+      <osmand:address>Brak określonego adresu</osmand:address>
+      <osmand:icon>military_landuse</osmand:icon>
+      <osmand:background>circle</osmand:background>
+      <osmand:color>#00ff00</osmand:color>
+      <osmand:amenity_origin>Amenity:Jednostka Wojskowa Nr 4393 i 4395: military:military_landuse</osmand:amenity_origin>
+    </extensions>
+  </wpt>
+  <wpt lat="52.2136024" lon="19.4585842">
+    <ele>111</ele>
```

Comment:
I noticed that we don't use ele for gpx, but looks like there is some place for it in kml: kml::ParsePointWithAltitude. I can raise an issue to support it in gpx parser, but how do we use it - I was not able to find it in OM?

Reply:
It may not be used yet, but we need to support it in the future to get track info/stats and to export back the important elevation data. A separate issue would be great.

Reply:
#5406 is added